kubectl-rook-ceph.sh

```diff
@@ -493,6 +493,42 @@ function run_debug() {
     ;;
   esac
 }
+
+function run_dr_clusters_health() {
+  [[ -z "${1:-""}" ]] && fail_error "Missing subcommand"
+  subcommand="$1"
+  shift # remove the subcommand from the front of the arg list
+  case $subcommand in
+  health)
+    run_dr_health "$@"
+    ;;
+  *)
+    fail_error "'dr' subcommand '$subcommand' does not exist"
+    ;;
+  esac
+}
+
+function run_dr_health() {
+  # do not call end_of_command_parsing here because all remaining input is passed directly to 'ceph'
+  info_msg "list of all the blockpools: "
+  KUBECTL_NS_CLUSTER get cephblockpool | awk '{print $1}' | sed "1 d"
+  echo
+  info_msg "enter ceph cephblockpool name: "
+  read -r resource_name
+  [ -z "$resource_name" ] && error_msg "ceph cephblockpool name can't be empty"
+  # `sed -e 's/^.//' -e 's/.$//'` is used to remove first and last char. It is used twice below to remove `["` form start and `"]` from end.
+  SECRET_NAME=$(KUBECTL_NS_CLUSTER get cephblockpool "$resource_name" -ojsonpath='{.spec.mirroring.peers.secretNames}' | sed -e 's/^.//' -e 's/.$//' | sed -e 's/^.//' -e 's/.$//')
+  MON_HOST=$(KUBECTL_NS_CLUSTER get secret/"$SECRET_NAME" -o jsonpath='{.data.token}' | base64 --decode | base64 --decode | jq '.mon_host' | sed -e 's/^.//' -e 's/.$//' | sed -e 's/^.//' -e 's/.$//')
+  PEER_KEY=$(KUBECTL_NS_CLUSTER get secret/"$SECRET_NAME" -o jsonpath='{.data.token}' | base64 --decode | base64 --decode | jq '.key' | sed -e 's/^.//' -e 's/.$//')
+  # run ceph status command form one cluster to another
+  echo
+  info_msg "running ceph status from other cluster details"
+  run_ceph_command -s --mon-host "$MON_HOST" --id rbd-mirror-peer --key "$PEER_KEY" "$@"
+  # run rbd pool mirror status
+  info_msg "running mirroring daemon health"
+  run_rbd_command -p "$resource_name" mirror pool status
```

Comment:
this is local pool status right not the remote cluster pool status

Reply:
right.

Reply:
AFAIK it should be local pool status. If the `ceph status` result fails to return from opposite cluster it means Ceph communication from local cluster -> remote cluster is not working and this should cause local cluster mirror `daemon` health to be in `WARNING` or `Error`.